app/controllers/users_controller.rb

```diff
@@ -38,7 +38,7 @@ def create
 
     respond_to do |format|
       if @user.save
-        format.html { redirect_to params[:redirect_to] || @user, notice: 'User was successfully created.' }
+        format.html { redirect_to params[:redirect_to].blank? ? @user : params[:redirect_to], notice: 'User was successfully created.' }
```

Comment:
Nice!! I did not know this method... really cool. I saw that the parameter is not nil, but it is empty. I will fix it now. Thanks!